src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -53,6 +53,10 @@ def handle_exception(ex):  # pylint: disable=too-many-return-statements
             return 1
         if isinstance(ex, (CLIError, CloudError, AzureException)):
             logger.error(ex.args[0])
+            try:
+                logger.error(ex.args[0].error.details[0])
+            except (AttributeError, IndexError):
+                pass
```

Comment:
I'd suggest to iterate `details`. If `details` is `None`, we catch `TypeError`. `IndexError` will not be triggered because `for` loop will not be executed for `[]`.

```suggestion
            try:
                for detail in ex.args[0].error.details:
                    logger.error(detail)
            except (AttributeError, TypeError):
                pass
```